test/functionalTest/cases/1585_log_summary/log_summary.test

```diff
@@ -176,7 +165,7 @@ Date: REGEX(.*)
  Transactions: 4 (new: 4)
  DB status: ok (created: 0, removed: 0)
  Notification failure active alarms: 0 (created: 0, removed: 0)
- Bad input active alarms: 1 (created: 6, removed: 0)
+ Bad input active alarms: 1 (created: 1, removed: 0)
```

Comment:
I mean that the echo message was wrong:

"06. grep SUMMARY in log-file (4 transactions, 4 bad input)"

However, this has changed after the new line format, so let's close this comment thread and see how it is now (I will add new comments if needed).

NTC